egress/tests/test_numeric.py

```diff
@@ -65,3 +69,17 @@ def test_insert_numeric_ok(self):
         """
         result = self.execute_operation_on_db(insert_numeric)
         self.assertEqual(result, 1)
+
+    def test_integer_array(self):
+        insert_array = """
+        INSERT INTO Bookings (num_of_bookings_per_month) VALUES (ARRAY[1, 2])
```

Comment:
Where do you test _selecting_ the value to see if we can decode it?

You're not testing your decoder at all. If it did you would have discovered parse_anyarray is incomplete.